themes/bootstrap3/templates/notifications/headerbroadcasts.phtml

```diff
@@ -0,0 +1,13 @@
+<?php
+  $broadcasts = $this->notifications()->getBroadcasts(true);
+?>
+<div class="broadcast_wrapper_global">
+  <?php foreach($broadcasts as $broadcast_id => $broadcast): ?>
+    <span class="broadcast" style="background-color:<?=$broadcast['color_value']?>; border-color:<?=$broadcast['border_color_value']?>;">
+      <a class="close-broadcast" href="#" data-broadcast-id="<?=$broadcast_id?>" title="<?=$this->transEsc('Close for this session')?>">
```

Comment:
I notice that there's a new `Close for this session` language string introduced here, but there doesn't seem to be an entry added to en.ini. Are there any other new strings we need to account for? It would be helpful to add all new strings to en.ini so that they can be picked up by our translation processes and eventually added to all of the other languages as well.

Reply:
Yes, there are several translations. I will check all the text again for consistency and add the default translation to en.ini and de.ini.

Reply:
The translations are added to the language files and checked for consistency.